ocw-course-v2/ocw-studio.yaml

```
@@ -832,6 +832,127 @@ collections:
                 Education Policy: []
                 Educational Technology: []
                 Higher Education: []
+          - label: MIT Learn Topics
+            name: mit_learn_topics
+            widget: hierarchical-select
+            levels:
+              - { label: "Topic", name: "topic" }
+              - { label: "Subtopic", name: "subtopic" }
+            max: 2
+            help: >
+              This field allows MIT Learn content curators to apply
+              course topics in learn.mit.edu.
+              Only edit if you are an MIT Learn content curator.
```

```suggestion
              Only MIT Learn content curators should edit this. 
```

Can this help text be moved closer to the field label? I guess that would require a change in ocw-studio.